lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -843,20 +846,27 @@ def lookup_country_code(country_field)
         country_code&.code(:alpha2)
       end
 
-      def add_stored_credential_options(xml, options={})
+      def add_stored_credential_subsequent_auth(xml, options={})
```

Comment:
couldn't really think of a better way to refactor this, so any other ideas welcome!